flytekit/core/data_persistence.py

```diff
@@ -248,7 +248,10 @@ def get(self, from_path: str, to_path: str, recursive: bool = False, **kwargs):
                     self.strip_file_header(from_path), self.strip_file_header(to_path), dirs_exist_ok=True
                 )
             print(f"Getting {from_path} to {to_path}")
-            return file_system.get(from_path, to_path, recursive=recursive, **kwargs)
+            dst = file_system.get(from_path, to_path, recursive=recursive, **kwargs)
```

Comment:
what does dst stand for?

Reply:
maybe add a test like https://github.com/flyteorg/flytekit/blob/38c76876dfe7fc2c62536ca6a195bce8a56c6270/tests/flytekit/unit/core/test_type_engine.py#L203, just making a dummy transformer, if that's easy to do.  if not, just add the below as a comment in both places.

Reply:
yes, since fsspec uses dst as well